packages/amplify-util-mock/src/__tests__/func/multi-function.test.ts

```diff
@@ -0,0 +1,49 @@
+import { prompter } from '@aws-amplify/amplify-prompts';
+import { start as lambdaServerStart } from '../../func';
+import { $TSAny } from 'amplify-cli-core';
+
+jest.mock('../../func');
+jest.mock('@aws-amplify/amplify-prompts');
+jest.mock('amplify-cli-core', () => ({
+  ...jest.requireActual('amplify-cli-core'),
+  FeatureFlags: {
+    getNumber: jest.fn(),
+  },
+}));
+
+const prompter_mock = prompter as jest.Mocked<typeof prompter>;
+prompter_mock.pick.mockResolvedValue(['func1', 'func2', 'func3']);
+const lambdaServerStartMock = lambdaServerStart as jest.MockedFunction<typeof lambdaServerStart>;
+
+describe('multiple function start', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const context_stub: $TSAny = {
+    input: {
+      options: {
+        event: 'event.json',
+        timeout: 1,
+      },
+    },
+    amplify: {
+      inputValidation: () => () => true,
+      readJsonFile: jest.fn(),
+      getResourceStatus: () => ({ allResources: [] }),
+      getEnvInfo: () => ({ envName: 'testing' }),
+      getMeta: () => ({
+        function: {
+          func1: {},
+          func2: {},
+          func3: {},
+        },
+      }),
+    },
+  };
+
+  it('prompts for multiple functions and selects the resource name', async () => {
+    await lambdaServerStart(context_stub);
+    expect(lambdaServerStartMock).toBeCalledTimes(1);
```

Comment:
What is this test testing?

Reply:
checking if the lambdaserver has been called , added additional expects to check if the function invoker has been called for each function.

Reply:
That would make sense if you were calling `lambdaServerStart` indirectly. I.e. 
if your call stack would look like `test -> higher level abstaction -> lambdaServerStart` . If you're calling into `lambdaServerStart` which is at this point mocked implementation then this test doesn't execute any logic.

Reply:
removed the `multi-function.test.ts` file and added the test to `index.test.ts` as we do have current test to call a single function on project with multiple functions. added test that calls multiple functions in a project improving on the existing test.